var/spack/repos/builtin/packages/rust/package.py

```diff
@@ -56,9 +54,7 @@ class Rust(Package):
         description='Install Rust source files'
     )
     variant(
-        'extra_targets',
-        default=(),
-        multi=True,
+        'extra_targets', default='none', multi=True,
```

Comment:
@westernmagic I guess nsimd wasn't a good example. :)

Reply:
thought so from the documentation, but deferred to your expertise ;)